test/integration/provider/gcp.go

```diff
@@ -92,8 +91,21 @@ func getInstancesWithTag(ctx context.Context, svc *compute.Service, searchTagNam
 	return instancesID, nil
 }
 
-// getAvailableVolumes describes volumes with the specified tag
-func getAvailableVolumes(ctx context.Context, svc *compute.Service, tagName string, tagValue string, machineClass *v1alpha1.MachineClass, secretData map[string][]byte) ([]string, error) {
+//TerminateInstance terminates a specified instance
+func TerminateInstance(svc *compute.Service, project, zone, instanceName string) {
+	operation, err := svc.Instances.Delete(project, zone, instanceName).Context(context.Background()).Do()
+	if err != nil {
+		fmt.Printf("can't terminate the instance %s, %s\n", instanceName, err.Error())
+	}
+
+	err = providerDriver.WaitUntilOperationCompleted(svc, project, zone, operation.Name)
+	if err != nil {
+		fmt.Printf("Deletion of volume %s failed with error: %s\n", instanceName, err.Error())
```

Comment:
Shouldn't the error message be 
```suggestion
		fmt.Printf("Deletion of instance %s failed with error: %s\n", instanceName, err.Error())
```